src/use-document-query-command-state.hook.js

```diff
@@ -20,7 +20,16 @@ export function useDocumentQueryCommandState(commandName) {
   function recheckActive() {
     const isActuallyActive = document.queryCommandState(commandName)
     const actualActiveValue = document.queryCommandValue(commandName)
-    if (isActuallyActive !== activeInfo.isActive || actualActiveValue !== activeInfo.value) {
+    if (tagName) {
+      const newIsActive = tagName === actualActiveValue // 'h1' === 'h1'
+      if (newIsActive !== activeInfo.isActive || actualActiveValue !== activeInfo.value) {
```

Comment:
Instead of copy pasting this if statement and create an if/else that is mostly the same, I think it could be easier to do this just with a ternary:

```js
if (isActuallyActive !== activeInfo.isActive || actualActiveValue !== activeInfo.value) {
  setActiveInfo({
    isActive: activeValueMatch ? actualActiveValue === activeValueMatch : isActuallyActive,
    value: actualActiveValue,
  })
}
```